sycl/doc/extensions/proposed/sycl_ext_oneapi_cuda_cluster_group.asciidoc

```diff
@@ -0,0 +1,326 @@
+= sycl_ext_oneapi_cuda_cluster_group
+
+:source-highlighter: coderay 
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+:endnote: &#8212;{nbsp}end{nbsp}note
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks] 
+
+Copyright (C) 2024-2024 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of
+The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+Other company and product names may be trademarks of the respective companies
+with which they are associated and can be claimed as the property of others.
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 8 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision. 
+
+This extensions also depends on the following other sycl extensions: 
+
+* link:../experimental/sycl_ext_oneapi_enqueue_functions.asciidoc[
+          sycl_ext_oneapi_enqueue_functions]
+* link:../experimental/sycl/sycl_ext_oneapi_properties.asciidoc[
+    sycl_ext_oneapi_properties
+]
+
+
+== Status
+
+This is an experimental extension specification for the `ext_oneapi_cuda`
+backend, intended to provide early access to features and gather community
+feedback.  
+Interfaces defined in this specification are implemented in {dpcpp}, but they
+are not finalized and may change incompatibly in future versions of {dpcpp}
+without prior notice. *Shipping software products should not rely on APIs
+defined in this specification.*
+
+=== Backend support status 
+
+This extension is only supported by the `ext_oneapi_cuda` backend.
```

Comment:
We should have this say the extension is not implemented, and then update this once the implementation PR has been merged, just to avoid this being incorrect due to PR merge order.

Reply:
Fixed